mne/viz/evoked.py

```diff
@@ -2225,6 +2225,9 @@ def plot_compare_evokeds(evokeds, picks=None, colors=None,
     if show_sensors is None:
         show_sensors = (len(picks) == 1)
 
+    if not callable(combine):
+        _validate_type(combine, (None, str), 'combine',
+                       'None, str or callable')
```

Comment:
Don't we need to test / allow for `callable` here, too?

Reply:
@larsoner How would one check for `callable` using `_validate_type`?

Reply:
@matteoanelli I just talk to @larsoner and we'll most probably want to enhance `_validate_type()` to accept `callable`. We should do that in a separate PR I would suggest.

Let me know if/when you're available and we can look into this together. otherwise I can start working on it on my own. Thanks!

Reply:
Oh it actually seems that adding the string `'callable'` might be sufficient! I will test this now.

Reply:
I tested the function and it looks like it is working with 'callable' as a string. However, it may be a good thing to add an explanation of what 'item_name', 'type_name' do inside '_validate_type()'.